examples/demo/advanced/scalar_image_function_inspector_old.py

```diff
@@ -123,7 +123,7 @@ class PlotUI(HasTraits):
     _image_index = Instance(GridDataSource)
```

Comment:
This example seems to be broken. When I run the example, I see the following error. This doesn't need to be fixed in this PR - but an issue (if one doesn't exist already) would be nice

```
python scalar_image_function_inspector_old.py
Traceback (most recent call last):
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\qt4\base_window.py", line 213, in paintEvent
    self.handler.paintEvent(event)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\qt4\base_window.py", line 65, in paintEvent
    self._enable_window._paint(event)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\abstract_window.py", line 471, in _paint
    self.component.draw(gc, view_bounds=(0, 0, size[0], size[1]))
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\component.py", line 427, in draw
    self._draw(gc, view_bounds, mode)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\component.py", line 779, in _draw
    self._dispatch_draw(layer, gc, view_bounds, mode)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\container.py", line 273, in _dispatch_draw
    component._dispatch_draw(layer, gc, new_bounds, mode)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\container.py", line 271, in _dispatch_draw
    component._draw(gc, new_bounds, mode)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\component.py", line 769, in _draw
    self._dispatch_draw(layer, bb, view_bounds, mode)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\container.py", line 273, in _dispatch_draw
    component._dispatch_draw(layer, gc, new_bounds, mode)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\enable\component.py", line 799, in _dispatch_draw
    handler(gc, view_bounds, mode)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\chaco\base_2d_plot.py", line 204, in _draw_image
    self._render(gc)
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\chaco\contour_poly_plot.py", line 46, in _render
    self._update_polys()
  File "C:\Users\rporuri\.edm\envs\chaco-test-3.6-pyqt5\lib\site-packages\chaco\contour_poly_plot.py", line 86, in _update_polys
    c = Cntr(xg, yg, data, ~mask)
ValueError: Arguments x, y, z, mask (if present) must have the same dimensions.
```

Reply:
Opened #558